pygmt/base_plotting.py

```diff
@@ -816,6 +816,175 @@ def plot(self, x=None, y=None, data=None, sizes=None, direction=None, **kwargs):
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("plot", arg_str)
 
+    @fmt_docstring
+    @use_alias(
+        A="straight_line",
+        B="frame",
+        C="cmap",
+        D="offset",
+        G="color",
+        I="intensity",
+        J="projection",
+        Jz="zscale",
+        JZ="zsize",
+        L="close",
+        N="no_clip",
+        Q="no_sort",
+        R="region",
+        S="style",
+        V="verbose",
+        W="pen",
+        X="xshift",
+        Y="yshift",
+        i="columns",
```

Comment:
I've aliased columns (i) here to make it consistent with `plot`. Let me know if this is a good idea, or if we should remove this alias as suggested in the discussion in https://github.com/GenericMappingTools/pygmt/pull/666#discussion_r511538159.

Reply:
Let's not add it the plot3d.

Reply:
I just saw that you also used *columns* in the tests. Perhaps it's OK to add it, and then break the backward compatibility with **plot()**.

Reply:
Sounds good.